networking/using-ptp.adoc

```diff
@@ -25,6 +25,11 @@ You can use the {product-title} console or `oc` CLI to install PTP by deploying
 
 include::modules/nw-ptp-introduction.adoc[leveloffset=+1]
 
+[IMPORTANT]
+====
+Before enabling PTP, ensure that NTP is disabled for the required nodes. xref:../post_installation_configuration/machine-configuration-tasks.adoc#cnf-disable-chronyd_post-install-machine-configuration-tasks[You can disable the NTP chrony time service (`chronyd`) using a MachineConfig CR].
```

Comment:
* I'd suggest to put "You can" outside of the link text. 
* Also consider whether the "using a MachineConfig CR" part is necessary or if it could be removed. If you do keep it, it should have backticks around `MachineConfig`.